pom.xml

```diff
@@ -323,6 +323,11 @@
             <artifactId>commons-io</artifactId>
             <version>2.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
```

Comment:
Was commons-compress added intentionally?

Reply:
Yes. In the build info code, we're using decompression: https://github.com/jfrog/build-info/blob/master/build-info-extractor-npm/src/main/java/org/jfrog/build/extractor/npm/extractor/NpmPublish.java#L99
Without it I'm getting this error:
```
Error: Unable to initialize main class org.jfrog.build.extractor.npm.extractor.NpmPublish
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/compress/compressors/gzip/GzipCompressorInputStream
```